tests/nightly/nvidia-drivers-compat/task.yaml

```diff
@@ -0,0 +1,39 @@
+summary: Collect and check nvidia driver libraries
+
+details: |
+  The test collects information about userspace libraries shipped by various
+  nvidia driver releases across a number of Ubuntu releases.
+
+# it's enough to run on a single host as the test launches all other releases in
+# containers
+
+systems: [ubuntu-24.04-64]
+
+kill-timeout: 60m
+
+execute: |
+  snap install lxd
+  lxd init --auto
+  lxd waitready
+
+  # TODO include Debian?
+  ./collect-release.sh 18.04
```

Comment:
This could move to a variant so that the test can run faster across more machines. You'd have to adjust the diff logic to handle per-release data though.

Reply:
that's true, but collection and checking if the global list of all files would become unnecessarily more complicated if it needs to consider part of the global list. Besides, it's a nightly test that finishes under 15 minutes.

Reply:
That's okay, let's skip this